### ISSUE TITLE
새로 업데이트 된 코드를 사용해도 하얀색 나오네요

### ISSUE THREAD

**Issue by salgilbarana** [bjpublic/Reactnative#49]
윗분과 같은 현상으로 나오네요
url도 최신으로 변경된걸로 해도 같은 증상이 나와요

<img width="1151" alt="스크린샷 2021-05-13 오후 3 55 38" src="https://user-images.githubusercontent.com/39834903/118089954-e2ecbf80-b403-11eb-9f4d-b27fe5fbe58b.png">

**Comment by dev-yakuza** [comment]
@salgilbarana 
안녕하세요.

우선 불편을 드려서 죄송합니다.
혹시 이 https://github.com/bjpublic/Reactnative/issues/12#issue-600689582 링크에 내용처럼 `src/Context/RandomUserData/index.tsx` 파일을 수정해 보셨나요?
수정하셔도 계속 동일한 문제가 발생하시는 건가요?

**Comment by salgilbarana** [comment]
> @salgilbarana
> 안녕하세요.
> 
> 우선 불편을 드려서 죄송합니다.
> 혹시 이 [#12 (comment)](https://github.com/bjpublic/Reactnative/issues/12#issue-600689582) 링크에 내용처럼 `src/Context/RandomUserData/index.tsx` 파일을 수정해 보셨나요?
> 수정하셔도 계속 동일한 문제가 발생하시는 건가요?

try {
      const response = await fetch(
        'https://raw.githubusercontent.com/dev-yakuza/users/master/api.json',
      );
      const data = await response.json();
      setUserList(data);
      setCachedData('UserList', data);
    } catch (error) {
      console.log('1',error);
    }
이거 말씀이시죠 이거대로 적용하는데도 하얀색 만 나오네요

**Comment by dev-yakuza** [comment]
@salgilbarana 
확인해보도록 하겠습니다.

**Comment by salgilbarana** [comment]
> @salgilbarana
> 확인해보도록 하겠습니다.

정말 감사합니다. 이렇게 빨리 답변 주시는분은 처음이라
제가 개인적으로 코드 따라가서보니

https://opinionated-quotes-api.gigalixirapp.com/v1/quotes?rand=t&n=25부분에서 값을 못받아오는것 같은데요

**Comment by dev-yakuza** [comment]
@salgilbarana 
확인까지 해주시고 감사합니다.
말씀해주신 링크에서 사진 설명들을 가져오는데, 지금 링크를 들어가보니, 링크가 동작하지 않는거 같습니다. 일시적인건지 서비스가 종료된건지 확인이 안되네요 😭 

사용자처럼 제가 임시로 데이터를 만들어야 할거 같습니다. 조금 시간이 걸릴거 같네요. 😭

**Comment by salgilbarana** [comment]
> @salgilbarana
> 확인까지 해주시고 감사합니다.
> 말씀해주신 링크에서 사진 설명들을 가져오는데, 지금 링크를 들어가보니, 링크가 동작하지 않는거 같습니다. 일시적인건지 서비스가 종료된건지 확인이 안되네요 😭
> 
> 사용자처럼 제가 임시로 데이터를 만들어야 할거 같습니다. 조금 시간이 걸릴거 같네요. 😭

ㅎㅎ 아닙니다. 감사합니다. 제가 임의 데이터 만들어서 해볼게요 ㅎㅎ

**Comment by dev-yakuza** [comment]
@salgilbarana 
제가 아직 확인해보지는 못했는데

https://baconipsum.com/api/?type=meat-and-filler&paras=25

위 링크로 교체하시면 일단 프로젝트는 실행할 수 있지 않을까 싶습니다.

**Comment by salgilbarana** [comment]
> @salgilbarana
> 제가 아직 확인해보지는 못했는데
> 
> https://baconipsum.com/api/?type=meat-and-filler&paras=25
> 
> 위 링크로 교체하시면 일단 프로젝트는 실행할 수 있지 않을까 싶습니다.

제가 하드 코딩으로 text 배열에 임의의 데이터를 넣어서 하니까 잘되네요
주신 url로 fetch 함수를 사용하려면 아래 

```javascript

      let text = [];
      for (const index in data.quotes) {
        text.push(data.quotes[index].quote);
      }

      setDescriptionList(text);
      setCachedData('DescriptionList', text);
```

수정이 필요한거 아닌가요 데이터를 저장 못하던데
23개에서 멈추는것 같고..

**Comment by dev-yakuza** [comment]
@salgilbarana
확인 감사합니다. 역시 제대로 임시 데이터를 만들어놔야겠네요.
하드 코딩으로 해결하셨다니 정말 다행이네요 😭

**Comment by salgilbarana** [comment]
> @salgilbarana
> 확인 감사합니다. 역시 제대로 임시 데이터를 만들어놔야겠네요.
> 하드 코딩으로 해결하셨다니 정말 다행이네요 😭

네 ㅎㅎㅎ 리액트 네이티브 이제 한달 정도되는데  저자님 책과 블로그로  많은 도움이 되고 있습니다.  항상 좋은 코드 기대하겠습니다.

**Comment by dev-yakuza** [comment]
@salgilbarana 
지금도 잘 하고 계실거 같아 큰 도움이 안될거 같지만, 지적해 주신 부분을 수정한 내용을 공유해 드립니다.

https://github.com/dev-yakuza/Reactnative/blob/bb48816b79b9fe6e090215c9ab77394b4c45f62a/ch9_SNSApp/src/Context/RandomUserData/index.tsx#L45-L62

위에 소스가 참고가 되었으면 좋겠습니다.
다시 한번 불편을 드려서 죄송합니다 🙏 

다른 분들도 이 이슈가 도움이 되었으면 좋겠네요!

책을 거의 다 끝내신거 같습니다! 올해에는 @salgilbarana 님이 만드신 앱을 써보는 날이 오면 좋을거 같습니다!
감사합니다 🙇‍♂️
